xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1566,6 +1567,15 @@ bool CPVRClients::OpenStream(const CPVRRecordingPtr &channel)
       client->OpenStream(channel))
   {
     CSingleLock lock(m_critSection);
+
+    CDateTime endTime = channel->RecordingTimeAsLocalTime() +
+        CDateTimeSpan(0, 0, channel->GetDuration() / 60, channel->GetDuration() % 60);
+
+    m_bIsRecordingInProgress = (endTime > CDateTime::GetCurrentDateTime());
```

Comment:
we have buffering for both but live streams require a different kind of buffering